src/maintainers.sh

```diff
@@ -160,9 +160,12 @@ function print_files_authors()
   local printed_authors_separator=false
 
   for file in "${files[@]}"; do
-    authors=$(grep --only-matching --extended-regexp 'MODULE_AUTHOR *\(.*\)' "$file" |
-      sed --regexp-extended 's/(MODULE_AUTHOR *\( *\"|\" *\))//g' |
-      sed --expression ':a' --expression 'N' --expression '$!ba' --expression 's/\n/, /g')
+    authors=$(grep --null-data --perl-regexp --only-matching 'MODULE_AUTHOR *\(\K[^)]*' "$file" |
+      tr -d '\0' |
+      sed --expression ':a' --expression 'N' --expression '$!ba' |
+      sed --expression 's/\s*\\//g' --expression 's/^\s*/ /g' |
+      tr -d '\n' |
+      sed --expression 's/\"\s*\"/, /g' --expression 's/\s*\"//g' --expression 's/\s*$//g')
```

Comment:
Alright, this is a huge command, could you help me to unpack it?

First: Could you elaborate a little bit about the `\K` option in this context? Maybe you can consider adding some comments to make it easier to understand these conditions.

Second: Could you break this huge command into a small set of commands? It is hard to review all of these in a single line.

Also, check this code style: https://kworkflow.org/content/codingstyle.html#avoid-short-options

I also noticed that when the MODULE_AUTHOR has `\n,` your change just drops the `\.` Check my comment in the test part.

Reply:
Made a comment regarding this. Hope now it's easier to understand what is happening